pkg/sdk/testint/streamlits_integration_test.go

```diff
@@ -0,0 +1,148 @@
+		require.Equal(t, "ROLE", e.OwnerRoleType)
+	}
+
+	t.Run("create streamlit", func(t *testing.T) {
+		stage, cleanupStage := createStage(t, client, databaseTest, schemaTest, random.AlphaN(4))
+		t.Cleanup(cleanupStage)
+
+		comment := random.StringN(4)
+		id := sdk.NewSchemaObjectIdentifier(databaseTest.Name, schemaTest.Name, random.StringN(4))
+		mainFile := "manifest.yml"
+		request := sdk.NewCreateStreamlitRequest(id, stage.Location(), mainFile).WithComment(&comment)
+		err := client.Streamlits.Create(ctx, request)
+		require.NoError(t, err)
+		t.Cleanup(cleanupStreamlitHandle(id))
+
+		assertStreamlit(t, id, comment, "")
+	})
+
+	t.Run("alter streamlit: set", func(t *testing.T) {
+		stage, cleanupStage := createStage(t, client, databaseTest, schemaTest, random.AlphaN(4))
+		t.Cleanup(cleanupStage)
+		manifest := "manifest.yml"
+		e := createStreamlitHandle(t, stage, manifest)
+
+		id := sdk.NewSchemaObjectIdentifier(databaseTest.Name, schemaTest.Name, e.Name)
+		comment := random.StringN(4)
+		set := sdk.NewStreamlitSetRequest(sdk.String(stage.Location()), &manifest).WithComment(&comment)
+		err := client.Streamlits.Alter(ctx, sdk.NewAlterStreamlitRequest(id).WithSet(set))
+		require.NoError(t, err)
+		assertStreamlit(t, id, comment, "")
+	})
+
+	t.Run("alter function: rename", func(t *testing.T) {
+		stage, cleanupStage := createStage(t, client, databaseTest, schemaTest, random.AlphaN(4))
+		t.Cleanup(cleanupStage)
+		e := createStreamlitHandle(t, stage, "manifest.yml")
+
+		id := sdk.NewSchemaObjectIdentifier(databaseTest.Name, schemaTest.Name, e.Name)
+		nid := sdk.NewSchemaObjectIdentifier(databaseTest.Name, schemaTest.Name, random.StringN(3))
+		err := client.Streamlits.Alter(ctx, sdk.NewAlterStreamlitRequest(id).WithRenameTo(&nid))
+		if err != nil {
+			t.Cleanup(cleanupStreamlitHandle(id))
+		} else {
+			t.Cleanup(cleanupStreamlitHandle(nid))
+		}
+		require.NoError(t, err)
+
+		_, err = client.Streamlits.ShowByID(ctx, id)
+		require.ErrorIs(t, err, collections.ErrObjectNotFound)
+
+		o, err := client.Streamlits.ShowByID(ctx, nid)
+		require.NoError(t, err)
+		require.Equal(t, nid.Name(), o.Name)
+	})
+
+	t.Run("show streamlit: with like", func(t *testing.T) {
+		stage, cleanupStage := createStage(t, client, databaseTest, schemaTest, random.AlphaN(4))
+		t.Cleanup(cleanupStage)
+		e := createStreamlitHandle(t, stage, "manifest.yml")
+
+		streamlits, err := client.Streamlits.Show(ctx, sdk.NewShowStreamlitRequest().WithLike(&sdk.Like{Pattern: &e.Name}))
+		require.NoError(t, err)
+		require.Equal(t, 1, len(streamlits))
+		require.Equal(t, *e, streamlits[0])
+	})
+
+	t.Run("show streamlit: terse with like", func(t *testing.T) {
+		stage, cleanupStage := createStage(t, client, databaseTest, schemaTest, random.AlphaN(4))
+		t.Cleanup(cleanupStage)
+		e := createStreamlitHandle(t, stage, "manifest.yml")
+
+		streamlits, err := client.Streamlits.Show(ctx, sdk.NewShowStreamlitRequest().WithLike(&sdk.Like{Pattern: &e.Name}))
```

Comment:
Terse param is not set here, please set it and re-run the test to check.

Reply:
set terse, make sure that all values being set are set, and empy are empty
```
--- PASS: TestInt_Streamlits (1.06s)
    --- PASS: TestInt_Streamlits/show_streamlit:_terse_with_like (1.06s)
PASS
```